openshift/ci-operator/generate-ci-config.sh

```diff
@@ -229,7 +247,7 @@ EOF
 done
 }
 
-image_deps=$(generate_image_dependencies)
+image_deps="$(generate_image_dependencies)"
```

Comment:
Also touched this based on @devguyio comment here: https://github.com/openshift/knative-eventing/pull/1636#pullrequestreview-884128645